server/package.json

```diff
@@ -8,7 +8,9 @@
     "start": "gulp",
     "serve": "node ./build/app.js",
     "build": "gulp build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "generate-api": "node ./node_modules/@graphql-codegen/cli/dist/commonjs/cli.js --config graphql-gen.yml",
+    "prestart": "npm run generate-api"
   },
   "author": "Ryan Petschek",
```

Comment:
you should add yourself as primary author :)

Reply:
so people can hunt you down if there are bugs

Reply:
Done